app/services/index.js

```diff
@@ -1,2 +1,3 @@
-export { default as voteService } from './topics';
+import voteService from './topics';
```

Comment:
Reading Axel's [article](http://www.2ality.com/2014/09/es6-modules-final.html) about ES6 module syntax

> The default export is just another named export

I think for this reason, when we `export default expression`, we cannot immediately `import { foo } from 'module'` as `default` export is just another named export.

Personally, I like de-structuring in `import` statements as I find it easy to comprehend. 

Although `export default` does seem to be the recommended approach in [airbnb eslint](https://github.com/airbnb/javascript), I think it does rely on our design decisions. 

I think we should use `export default` when we have a module that does 1 thing, e.g. a Component, or a single utility function. In service, the intent behind it was to provide an interface to different service endpoints, I view `voteService` as a single utility function, exposed via the service interface.

What I'm getting is, I think in this case, it is alright to 
```js
export { default as voteService } from './topics';
```

without having a `export default`. I think having both a `export {}` and a `export default {}` might be more confusing to a user looking at this module and duplicates the code.

@ZeroCho What are your thoughts?